Controllers/StandUpController.js

```diff
@@ -1,207 +1,312 @@
+/* eslint-disable no-underscore-dangle */
 /* eslint-disable class-methods-use-this */
 import { validationResult } from 'express-validator';
+import { DateTime } from 'luxon';
```

Comment:
At the [moment site](https://momentjs.com/docs/#/-project-status/) they suggested that for new projects we could get on with other alternatives rather than using `moment`. From how they state it, the project is not really that supported anymore, and also how they say that the library size alone is big. This [post](https://inventi.studio/en/blog/why-you-shouldnt-use-moment-js) listed shows that `moment` is atleast 3x bigger than luxon in size..
`luxon` is a project that is kind of an [modernized](https://moment.github.io/luxon/#/why) DateTime library from `moment` that is why I went with `luxon` instead

Reply:
Great, thanks!